### ISSUE TITLE
[gop-it] add a config file

### ISSUE THREAD

**Issue by sontaimnt** [sontaimnt/gop-it#2]
in gop-it allow the user to edit config and instead of `RedString` they can use `BlueString` etc

# tasks:-
- [ ] implement json parsing
- [ ] changing theme
- [ ] json verification

**Comment by royadaneshi** [comment]
Hello 
Do you want the state of the game to be saved in a config.json?

**Comment by sontaimnt** [comment]
Yes, but my intention was to instead of using RedString in this json config users can assume custom color for their row
